recipes/xilinx-runtime/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "xilinx-runtime" %}
+{% set version = "2.13.466" %}
+{% set quarter = "202210" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Xilinx/XRT/archive/refs/tags/{{ quarter }}.{{ version }}.tar.gz
+  sha256: d73d8366f6f6d66bfdcb67e72135f559a4d95644c5416c27dea79c235953fd20
+  patches:
+    - conda.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  ignore_run_exports_from:
+    - libdrm
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - pkg-config
+    - git
+
+  host:
+    - python
+    - pybind11
+    - boost
+    - boost-cpp
+    - protobuf
+    - libprotobuf
+    - rapidjson
+    - libuuid
+    - libdrm
+    - clhpp
+    - libcurl
```

Comment:
The log output suggest that this isn't needed.
```suggestion
```

Reply:
@xhochy libcurl provides curl.h into the include/ directory which is required for building the C++. It may not be needed at runtime. Perhaps I should ignore run exports for libcurl just like I am for libdrm?

Reply:
Yes, sounds like a good approach!

Reply:
This should be resolved.